spec/controllers/grade_entry_forms_controller_spec.rb

```diff
@@ -566,4 +566,41 @@
       include_examples 'switch assignment tests'
     end
   end
+
+  describe '#populate_grade_table' do
```

Comment:
Please add a few more tests to make sure that the correct data is being sent, not just that *some* data is being sent.

Reply:
keys are fine :)